src/store/components/AdminItemPage/index.tsx

```diff
@@ -22,79 +23,115 @@ interface AdminItemPageProps {
 }
 
 interface AdminItemPageForm {
-  collection?: string;
-  itemName?: string;
-  description?: string;
+  collection: string;
+  itemName: string;
+  description: string;
   existingPicture?: string;
   newPicture?: Blob;
-  hidden?: boolean;
-  monthlyLimit?: number;
-  lifetimeLimit?: number;
-  hasVariantsEnabled?: boolean;
+  hidden: boolean;
+  monthlyLimit: string;
+  lifetimeLimit: string;
+  hasVariantsEnabled: boolean;
 
-  categoryName?: string;
-  options?: {
-    uuid?: Uuid;
+  categoryName: string;
+  options: {
+    uuid: string | undefined;
     value: string;
     price: string;
     quantity: string;
     discountPercentage: string;
   }[];
 
-  quantity?: string;
-  price?: string;
-  discountPercentage?: string;
+  quantity: string;
+  price: string;
+  discountPercentage: string;
 }
```

Comment:
Formik requires all fields to have a value at all times (including for initial values), so I removed the optionals and added the type to the `initialValues` object. The only two fields where this isn't the case are `existingPicture` and `newPicture`, which are uniquely controlled by an external component.